customManager/UtilsManager.ts

```
@@ -0,0 +1,71 @@
+/**
+ * @description 工具方法管理器
+ */
+class UtilsManager {
+  static instance: UtilsManager | null = null;
+
+  static getInstance(): UtilsManager {
+    if (!this.instance) {
+      this.instance = new UtilsManager();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)